config/jobs/kubernetes-sigs/lws/lws-presubmit.yaml

```diff
@@ -41,7 +41,7 @@ presubmits:
       description: "Run lws integration tests"
     spec:
       containers:
-      - image: public.ecr.aws/docker/library/golang:1.21
+      - image: public.ecr.aws/docker/library/golang:1.22
```

Comment:
I would expect that the unit tests would also need this.

Why are we using kubekins for our unit tests?